presto-product-tests/src/main/resources/sql-tests/datasets/table_with_header.data-generator

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Output 2 header lines
+echo 'id|data'
+echo 'INT|STRING'
+
+# loop outputs more than 32MB, while 32MB is size of a HDFS block
+for j in 0 `seq 31`; do
```

Comment:
apparently just over 32MB, right? Can you make it `seq 1 33`?

Reply:
ritght.